src/lib/bump-stats/types.ts

```diff
@@ -0,0 +1,29 @@
+export enum STATS_GROUP {
```

Comment:
```suggestion
export enum StatsGroup {
```
The nice thing about enums is that we get types and values kind of rolled into one. No need to export a separate `StatsGroup` type.